src/lib.rs

```diff
@@ -7,22 +7,34 @@
 #![deny(missing_docs)]
 extern crate libc;
 
+#[cfg(unix)]
 #[macro_use]
 pub mod ioctl;
 
+#[cfg(unix)]
 pub mod aio;
+#[cfg(unix)]
 pub mod errno;
+#[cfg(unix)]
 pub mod eventfd;
+#[cfg(unix)]
 pub mod fallocate;
 pub mod fam;
 pub mod file_traits;
+#[cfg(unix)]
 pub mod poll;
 pub mod rand;
+#[cfg(unix)]
 pub mod seek_hole;
+#[cfg(unix)]
 pub mod signal;
+#[cfg(unix)]
 pub mod sock_ctrl_msg;
+#[cfg(unix)]
 pub mod tempdir;
 pub mod tempfile;
+#[cfg(unix)]
 pub mod terminal;
+#[cfg(unix)]
```

Comment:
Could we group all the `#[cfg(unix)]` together in the same part of the `lib.rs` file, to make it clearer which modules are unix specific?

Reply:
Done

Reply:
I still think it doesn't look very nice as it is now. I had another idea in mind.

My proposal would be to create a new `unix` module and move all the unix specific utilities in that module. In `src/lib.rs` we would just export everything from unix only when on an unix platform.

In src/lib.rs we would just have:
```rust
#[cfg(unix)]
mod unix;

#[cfg(unix)]
pub use unix::*;

pub mod fam;
pub mod file_traits;
pub mod rand;
pub mod tempfile;
pub mod write_zeroes;
```

And in `src/unix/mod.rs`:
```rust
#[macro_use]
pub mod ioctl;
pub mod aio;
pub mod errno;
pub mod eventfd;
pub mod fallocate;
pub mod poll;
pub mod seek_hole;
pub mod signal;
pub mod sock_ctrl_msg;
pub mod tempdir;
pub mod terminal;
pub mod timerfd;
```

Nothing else needs to be changed and the vmm-sys-util exports the modules in the same way as before.

Reply:
> My proposal would be to create a new unix module and move all the unix specific utilities in that module

I thought about this but assumed this would force caller path inclusion changes. If there's a way to avoid that and move everything under a unix specific module, then I'm all for it.

Reply:
This line avoids the path inclusion: `pub use unix::*;`

Reply:
It's really a little ugly, agree to move them into a dedicate xxx_unix.rs file.